### ISSUE TITLE
Unaccessible Administration because of Mixed Content Block

### ISSUE THREAD

**Issue by dfsoeten** [shopware/platform#2170]
### PHP Version

7.4.25

### Shopware Version

6.4.5.1

### Expected behaviour

When Shopware is deployed behind a loadbalancer and served through HTTPS, the administration login should proceed.

### Actual behaviour

When Shopware is deployed behind a loadbalancer and served through HTTPS, the following error is thrown in `https://example.com/bundles/administration/static/js/vendors-node.js`:
```
Mixed Content: The page at 'https://example.com/admin#/login' was loaded over HTTPS, but requested an insecure XMLHttpRequest endpoint 'http://example.com/api/oauth/token'. This request has been blocked; the content must be served over HTTPS.
```

The `TRUSTED_PROXIES` environment variable is set to `REMOTE_ADDR` and `config/packages/<myenvironment>/framework.yaml` contains the following:
```
framework:
  trusted_headers: ['x-forwarded-proto', 'x-forwarded-for', 'x-forwarded-host', 'x-forwarded-port']
``` 
as per documentation [here](https://symfony.com/doc/current/deployment/proxies.html). 

I believe this is caused by misconfugration of the [`apiContext` in JavaScript](https://github.com/shopware/administration/blob/bf931584ebd34bb8536fa4fd6ce0b8705b98d631/Resources/views/administration/index.html.twig#L60):
```
<script nonce="">
        Shopware.Application.start({
        apiContext: {
            host: 'example.com',
            port: 80,
            scheme: 'http',
            schemeAndHttpHost: 'http://example.com',
            uri: 'http://example.com/admin',
            basePath: '',
            pathInfo: '/admin',
            liveVersionId: '0fa91ce3e96a4bc2be4bd9ce752c3425',
            systemLanguageId: '2fbb5fe2e29a4d70aa5854ce7ce3e20b',
            apiVersion: 4,
            assetPath: 'https://cdn.example.com'
        },
        ...
    });
</script>
```
However, when adding the following [after this line](https://github.com/shopware/production/blob/a4a7099f8c0ee259f2c7cfb92aba717b3cd5d50f/public/index.php#L75), it seems that Symfony is correctly configured:
```
dump($request->getScheme());
dump($request->getSchemeAndHttpHost());
exit();
```
which outputs: 
```
https
https://example.com
```
I'm running Shopware on a [Google Cloud Loadalancer](https://cloud.google.com/load-balancing) with [GKE](https://cloud.google.com/kubernetes-engine) as backend.

### How to reproduce

Add the following to `config/packages/<myenvironment>/framework.yaml`:
```
framework:
  trusted_headers: ['x-forwarded-proto', 'x-forwarded-for', 'x-forwarded-host', 'x-forwarded-port']
``` 

Set `TRUSTED_PROXIES` to `REMOTE_ADDR` and add the following to [this line](https://github.com/shopware/production/blob/a4a7099f8c0ee259f2c7cfb92aba717b3cd5d50f/public/index.php#L73) in `public/index.php`:
```
$_SERVER['HTTP_X_FORWARDED_PROTO'] = 'https';
$_SERVER['HTTP_X_FORWARDED_HOST'] = '127.0.0.1';
$_SERVER['HTTP_X_FORWARDED_FOR'] = '127.0.0.1';
```

Reload the page and inspect the `apiContext`. After add the following on [this line](https://github.com/shopware/production/blob/a4a7099f8c0ee259f2c7cfb92aba717b3cd5d50f/public/index.php#L75):
```
dump($request->getScheme());
dump($request->getSchemeAndHttpHost());
exit();
```

**Comment by dfsoeten** [comment]
Seems to be fixed in [0371ffc](https://github.com/shopware/platform/commit/0371ffc19de759ad7b3ce61f3f2c77d88f1df23b).

**Comment by keulinho** [comment]
Yep it should be fixed by the linked commit. 
The linked commit is included in the latest 6.4.6.0 release, so an upgrade should fix the issue.
Additionally there was already another issue regarding this: https://github.com/shopware/platform/issues/2099.

Nevertheless thanks for the detailed bug :bug:  description
